src/main/java/apoc/export/csv/CsvFormat.java

```diff
@@ -34,7 +34,7 @@
  */
 public class CsvFormat implements Format {
     private final GraphDatabaseService db;
-    private boolean quoteIfNeeded = true;
+    private boolean quoteIfNeeded = false;
```

Comment:
why was the the default changed?

Reply:
This was an extra flag to distinguish between _no-quotes_ and _quotes only if needed_. It's value is overwritten in the method `getCsvWriter`, de default is never used.
In the current implementation, however, the name is now misleading: there's only need to distinguish between _apply quotes to all_ and **not** _apply quotes to all_, so I changed the name to `applyQuotesToAll`